packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -176,7 +176,7 @@ export function initFlutterProjectWithProfile(cwd: string, settings: Object): Pr
   addCircleCITags(cwd);
 
   return new Promise((resolve, reject) => {
-    let chain = spawn(getCLIPath(), ['init'], { cwd, stripColors: true })
+    let chain = spawn(getScriptRunnerPath(), [getCLIPath(), 'init'], { cwd, stripColors: true })
```

Comment:
Would it be better to update spawn to use `process.execPath` as the executable rather than making this change everywhere we call spawn?

Reply:
I don't think so because unfortunately we do spawn other executables.

Reply:
perhaps then we should make a `spawnCLI()` function that just takes args to the CLI and all of the stuff about pathing is handled internally. Something like:
`spawnCli(['push', '--force'], {cwd, env, etc...})`
Then for spawning other processes or pointing to a different install of the CLI we can still use spawn(). But for 99% of use cases we are always going to call spawn with `getScriptRunnerPath(), [getCLIPath(), ...]` so it would be nice to have an abstraction around that

Reply:
This is a "nice to have" and there's already a tremendous amount of utility in this PR, so we could punt on this

Reply:
agreed - I'll make a note!